jib-cli/src/integration-test/java/com/google/cloud/tools/jib/cli/cli2/JarCommandTest.java

```diff
@@ -154,4 +167,71 @@ public void testJar_unknownMode() {
         .contains(
             "Invalid value for option '--mode': expected one of [exploded, packaged] (case-sensitive) but was 'unknown'");
   }
+
+  @Test
+  public void testSpringBootLayeredJar_explodedMode() throws IOException, InterruptedException {
+    springBootProjectLayered.build("clean", "bootJar");
+    Path jarParentPath =
+        springBootProjectLayered.getProjectRoot().toAbsolutePath().resolve("build").resolve("libs");
```

Comment:
Hmm... I'm curious. Does it not work without `toAbsolutePath()`?

Reply:
Ah good catch. `toAbsolutePath()` doesn't make a difference